### PR TITLE
Added wof:lang props for PR counties

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/507, related to a merged Pelias issue.

PR updates include:

- New `wof:lang` prop equal to `['spa']`
- New `wof:lang_x_spoken` prop equal to `['spa', 'eng']`
- New `wof:lang_x_official` prop equal to `['spa', 'eng']`
- Updated `wof:lastmodified` date

Additional updates are tracked in #290 and #511 for other locations/records.

### CODE REVIEW

File: data/102/080/271/102080271.geojson [hunk 1]
```diff
@@ -83,7 +83,18 @@
     ],
     "wof:id":102080271,
     "wof:label":"Municipio de Manat\u00ed",
-    "wof:lastmodified":1475267157,
+    "wof:lang":[
+        "spa"
+    ],
+    "wof:lang_x_official":[
+        "spa",
+        "eng"
```
Comment (nvkelso) [inline data/102/080/271/102080271.geojson]: I don't see English offered on their website:

- http://www.manati.pr

**Comment by nvkelso** [comment]
This Wikipedia article gives a good overview of language usage (official, spoken) in PR:

- https://en.wikipedia.org/wiki/English_language_in_Puerto_Rico

Summary:

- Across PR at a top level there are 2 official languages. 
- In practice some government functions are in English, some high schools are in English (either a foreign language class or most private school instruction). 
- In practice most people speak Spanish, especially away from the capital.

File: data/102/080/271/102080271.geojson (hunk 1 above)
Comment (nvkelso) [inline data/102/080/271/102080271.geojson]: _**for all:**_ It's technically not wrong to list `eng` here as _some_ official business and spoken language is in English, but dominant is Spanish (as recorded here with `"wof:lang" = "spa"`).

It is weird (but probably not technically wrong) to not include an English preferred name.

Comment (nvkelso) [inline data/102/080/271/102080271.geojson]: Something like: `Manati Municipality`.

**Comment by stepps00** [comment]
This was one of the websites I referenced when updating the language properties.. I chose to include English to err on the side of including more information and, like you pointed out, the 

`"...Puerto Rican government has declared English an official language on several occasions while removing it from that status on as many occasions. English remains an official language in the Commonwealth."`

I think the best approach here is to include English and Spanish, add add preferred names for all records in this PR (something like [this](https://github.com/whosonfirst-data/whosonfirst-data/pull/795#discussion_r117996736)).

**Comment by stepps00** [comment]
Most recent commit addressed all comments in this PR.

All records received:
- A new `name:eng_x_preferred` property equal to the record's `wof:name`
- A new `name:eng_x_variant` property equal to the record's `wof:name` + ' Municipality'

This workflow is consistent with past `name` property updates.

**Pull request merged**